src/components/windows-titlebar.tsx

```diff
@@ -6,6 +6,15 @@ import { cx } from '../lib/cx';
 import { getIpcApi } from '../lib/get-ipc-api';
 import Button from './button';
 
+interface HasTextInfo {
+	textInfo: { direction: 'ltr' | 'rtl' };
+}
+
+// If this breaks in a future Electron version, we might need to change `textInfo` to `getTextInfo()`
+// See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getTextInfo
+const isWindowFrameRtl =
```

Comment:
What prevents us from using `getTextInfo()` now?

Reply:
[`getTextInfo()` is coming in Chrome 130](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getTextInfo#browser_compatibility). Stable Chrome is currently version 129, let alone the version we use in Electron which is even older 😄

It's already in the TypeScript definitions though, so it's confusing for the code.

Reply:
Oh, thanks for the clarification. I saw a green mark near Chrome, so I thought it was already supported. It's confusing that they mark it as supported if it is added in a version that has yet to be released.